core/api/src/domain/quiz/sections.ts

```diff
@@ -0,0 +1,212 @@
+export const QuizzesSectionsConfig = [
```

Comment:
instead of having 2 sources of truth we need to add amounts to this `config`

Reply:
they represents different things:
- this represent which quiz are in which sections
- the other config represent the reward of each quiz

I can merge it but at the end of the day we get the same information, and I feel the config will be more complexed if merged